### ISSUE TITLE
error in running autogen.sh, for libsndfile #1

### ISSUE THREAD

**Issue by FreddyYJ** [Suresoft-GLaDOS/defects4cpp#50]
I got error in running `./autogen.sh`, in libsndfile 1 after checkout:
```
checking for autogen ... yes
checking for autoconf ... yes
checking for automake ... yes
checking for libtool ... libtoolize
checking for pkg-config ... yes
checking for python ... yes
Creating 'Cfg' directory.
Generating configuration files for libsndfile, please wait ... 
  aclocal -I M4
configure.ac:16: error: AC_CONFIG_MACRO_DIR can only be used once
./lib/autoconf/general.m4:1969: AC_CONFIG_MACRO_DIR is expanded from...
configure.ac:16: the top level
autom4te: error: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
```

**Comment by FreddyYJ** [comment]
But, libsndfile 2 works fine.

**Comment by FreddyYJ** [comment]
I found 1 and 3 throws this error, and 2 and 4 works fine.

**Comment by HansolChoe** [comment]
Thank you for your reporting @FreddyYJ
We fixed `libsndfile` issue (at 4d6c2ce3483663896e0f811d7d249afebbc97fde) . Please check these problem happens now.
Maybe you should rebuild the image 😃
